src/backend/distributed/executor/multi_router_executor.c

```diff
@@ -642,6 +662,61 @@ ExecuteSingleSelectTask(CitusScanState *scanState, Task *task)
 
 
 /*
+ * BuildPlacementSelectList builds a list of SELECT placement accesses
+ * which can be used to call StartPlacementListConnection or
+ * GetPlacementListConnection.
+ */
+List *
+BuildPlacementSelectList(char *nodeName, int nodePort, List *relationShardList)
```

Comment:
Added it to `GetModifyConnections()`, which is called by `ExecuteSingleModifyTask`